packages/node_modules/@webex/internal-plugin-encryption/src/kms.js

```diff
@@ -250,6 +251,147 @@ const KMS = WebexPlugin.extend({
       });
   },
 
+  /**
+  * @typedef {Object} FetchPublicKeyResponse
```

Comment:
in general i think these comments could be more descriptive. what are the possible `status` values? what is `requestId` used for? is it telemetry somewhere? etc.